pages/challenge.js

```diff
@@ -12,6 +12,32 @@ const RepoLink = 'https://github.com/OperationCode/front-end/';
 const ChallengeLink = `${RepoLink}blob/main/pages/challenge.js`;
 const CompareLink = `${RepoLink}compare`;
 
+export const MakeColumnsFromArray = ({ ary }) => {
+  const numberNamesPerColumn = 20;
+  const numberCols = Math.floor(ary.length / numberNamesPerColumn) + 1;
```

Comment:
Since you're using `challengers` (a variable defined outside the scope of this method), you've made an impure function (this sounds bad/negative, but there's no such connotation - it just is what it is). 

Since you've already made an impure, you can ditch the argument entirely, and just use `challengers.length`.